packages/react-router/package.json

```diff
@@ -77,6 +77,7 @@
     "jsdom": "^24.1.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "vitest": "^1.6.0",
```

Comment:
This shouldn't be needed, since Vitest gets inherited from the monorepo. Plus we don't have vitest as deps or devDeps in any of the other packages.

Its probably whats causing the lockfile failure in CI.

Reply:
Hmm. It was failing to run the tests locally without it.

Reply:
Oh, I see!!!

This tends to happen for me locally as well when I haven't kept up to date with the **lockfile** on _main_ or if my pnpm version isn't in sync with `packageManager` field in the `package.json` file.

Some common indicators of happening which I've noted are:
- local tests suddenly thinks vitest is missing
  - for which "fixing" it causes CI installs to fail
- local formatting using prettier causes unrelated files or areas of affected code to change their code-style.
- local builds failings with nx being unable to build one or more packages.

To fix this, I often switch to _main_ and pull down all the latest changes. Then make sure the pnpm version is correct and run `pnpm install`. Then for good measure, I run `pnpm dedupe`.